package.json

```diff
@@ -20,61 +20,61 @@
     "node": ">=12.22.6"
   },
   "dependencies": {
-    "address-rfc2821"       : "^2.0.1",
-    "address-rfc2822"       : "^2.1.0",
-    "async"                 : "~3.2.1",
-    "daemon"                : "~1.1.0",
-    "generic-pool"          : "~2.5.4",
-    "iconv"                 : "~3.0.1",
-    "ipaddr.js"             : "~2.0.1",
-    "libqp"                 : "^1.1.0",
-    "libmime"               : "^5.0.0",
-    "nopt"                  : "~5.0.0",
-    "npid"                  : "~0.4.0",
-    "semver"                : "~7.3.5",
-    "sprintf-js"            : "~1.1.2",
-    "haraka-config"         : "^1.0.20",
-    "haraka-constants"      : "^1.0.5",
-    "haraka-dsn"            : "^1.0.3",
-    "haraka-net-utils"      : "^1.3.2",
-    "haraka-notes"          : "^1.0.4",
-    "haraka-plugin-redis"   : "^1.0.12",
-    "haraka-results"        : "^2.0.3",
-    "haraka-tld"            : "^1.0.28",
-    "haraka-utils"          : "^1.0.2",
-    "openssl-wrapper"       : "^0.3.4",
-    "sockaddr"              : "^1.0.1"
+    "address-rfc2821": "^2.0.1",
```

Comment:
Those formatting changes are not intended, perhaps the linter or my IDE did it, I'm not sure.